From eaf8dfd0471bf1bf8fc5bd4b3f6b726db060a42a Mon Sep 17 00:00:00 2001
From: anoopsjohn <anoopsamjohn@gmail.com>
Date: Wed, 1 Jul 2015 11:26:24 +0530
Subject: [PATCH 241/295] HBASE-13670 [HBase MOB] ExpiredMobFileCleaner tool
 deletes mob files later for one more day after they
 are expired. (Jingcheng)

Reason: Bugfix
Author: Jingcheng Du
Ref: CDH-31077
---
 hbase-common/src/main/resources/hbase-default.xml |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/hbase-common/src/main/resources/hbase-default.xml b/hbase-common/src/main/resources/hbase-default.xml
index ba4a2a1..07b5018 100644
--- a/hbase-common/src/main/resources/hbase-default.xml
+++ b/hbase-common/src/main/resources/hbase-default.xml
@@ -1457,6 +1457,9 @@ possible configurations would overwhelm and obscure the important.
     <description>
       The period that ExpiredMobFileCleanerChore runs. The unit is millisecond.
       The default value is one day.
+      The MOB file name uses only the date part of the file creation time in it. We use this
+      time for deciding TTL expiry of the files. So the removal of TTL expired files might be
+      delayed. The max delay might be 24 hrs.
     </description>
   </property>
   <property>
-- 
1.7.9.5

